src/components/Radio/index.tsx

```diff
@@ -0,0 +1,112 @@
+import type { HTMLAttributes, ReactElement } from 'react'
+import styled from '@emotion/styled'
+
+export interface RadioProps extends HTMLAttributes<HTMLDivElement> {
+  formName: string
+  items: { code: string; name: string }[]
+  radioDirection: string
+  onChange(e: React.FormEvent<HTMLInputElement>): void
+}
+
+interface FormProps extends HTMLAttributes<HTMLDivElement> {
+  radioDirection: string
+}
+const Radio = ({
+  formName,
+  onChange,
+  items,
+  radioDirection = 'horizental',
+  ...props
+}: RadioProps): ReactElement => {
+  const handleRadiobutton = (e: React.FormEvent<HTMLInputElement>): void => {
+    onChange && onChange(e)
+  }
```

Comment:
```suggestion
  const handleRadiobutton = (e: React.FormEvent<HTMLInputElement>): void => {
    onChange(e)
  }
```
onChange를 필수값으로 받고 있어서 제거해줘도 좋을 것 같습니다!